src/main/java/seedu/address/model/event/EventSortFieldType.java

```diff
@@ -0,0 +1,24 @@
+package seedu.address.model.event;
+
+/**
+ * Represents the possible fields of an {@code Event} to sort by.
+ */
+public enum EventSortFieldType {
+
+    NAME("name"),
```

Comment:
This needs to be changed to reflect the event title attribute of Event class
```suggestion
    EVENT_TITLE("event title"),
```